packages/sdk/react-client/src/testing/withClientProvider.stories.tsx

```diff
@@ -0,0 +1,59 @@
+//
+// Copyright 2024 DXOS.org
+//
+
+import '@dxos-theme';
+
+import React from 'react';
+
+import { SyntaxHighlighter } from '@dxos/react-ui-syntax-highlighter';
+import { withLayout, withTheme } from '@dxos/storybook-utils';
+
+import { withClientProvider, withMultiClientProvider } from './withClientProvider';
+import { useClient } from '../client';
+import { type Space, useSpaces } from '../echo';
+
+const SpaceInfo = ({ space }: { space: Space }) => {
+  const name = space.isOpen ? space.properties.name : '';
+  return <SyntaxHighlighter language='json'>{JSON.stringify({ id: space.id, name }, null, 2)}</SyntaxHighlighter>;
+};
+
+const Test = () => {
+  const client = useClient();
+  // TODO(burdon): [SDK]: 'all' property has no effect.
```

Comment:
this todo should be on the actual sdk code

Reply:
i misread the usage. i thought this was to filter the default space.